### PR TITLE
(TM only) Add delay to telecomm and walkietalkie to sec locker

### PR DESCRIPTION
**Pull request by TheBoondock**
## About The Pull Request
This pr adds a delay to telecommunication, 10 seconds between send and broadcast. Walkietalkie remains untouched. Sec locker spawns with a walkietalkie to use as communication. 11am edit so i mightve done something wrong, please point it out for me if you see a mistake in the code. 
## Why It's Good For The Game
This is just an experiment, i want to see how this delay can change the meta. Now antag gets a little wiggle room to do their deeds before their target's cry for help is received.
## Changelog
:cl:

balance: Telecommunication now has a 10 seconds delay between sending a message and broadcasting
balance: Security locker now has a walkietalkie, they do not have a delay BE SURE TO SET A PRIVATE FREQUENCY
/:cl:

### CODE REVIEW

**Comment by Vect0r2** [comment]
Bob: Hey! Bill! Fuck you!
20 SECONDS
Bill: Imma beat ur ass, AI locate Bob the lizard
20 seconds later
HAL: BOB IS LOCATED IN CARGO
HAL: I MEAN HALLWAYS
HAL: I MEAN I CANNOT LOCATE THEM IN TIME
20 seconds later
Bob: What a bad change, I could just use a EMP for the same effect! It just requires me to 
Bob: *gasp
Bob: PREPARE!!

Also does this effect binary
I don't like it, gets rid of the fun banter that normal crew can have for "balance", even thou emp can do same thing.

**Comment by Singul0** [comment]
While the intention is good. limiting communications between crew severely fucks over general communications quite badly just to give antags a breathing room isn't a good trade-off.

**Comment by ChungusGamer666** [comment]
this is an ungodly change but i kind of want to see how this affects the game...

File: code/game/machinery/telecomms/machines/broadcaster.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@ GLOBAL_VAR_INIT(message_delay, 0) // To make sure restarting the recentmessages
 
 	if(!GLOB.message_delay)
 		GLOB.message_delay = TRUE
-		addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(end_message_delay)), 1 SECONDS)
+		addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(end_message_delay)), 10 SECONDS)
```
Comment (ChungusGamer666) [inline code/game/machinery/telecomms/machines/broadcaster.dm:51]: i don't think this does what you think it does

**Review by Fikou — CHANGES_REQUESTED** [review]
this pr does not function

**Comment by TheBoondock** [comment]
> this pr does not function

Its properly delayed now, i just need to solve some problem with message being gibberish() more than usual. Not sure why it has so much compression as i only delay the broadcasting not the signal creation on the headset

**Comment by TheBoondock** [comment]
On another note, i agree that 10 seconds really mess with regular conversation and since reduced it to 3 seconds for now. Sec walkietalkie also now spawn set to a specific frequency to have some privacy

**Pull request closed without merging**